src/components/Table/Table.tsx

```diff
@@ -20,10 +20,9 @@ interface CustomColors {
   color?: string;
 }
 
-export const tableBodyTextSizesArray = ["body1", "body2", "caption"] as const;
-export type TableBodyTextSizes = typeof tableBodyTextSizesArray[number];
-
 export interface TableProps extends BoxProps {
+  /** Additional class name for the `Table` component. */
+  className?: string;
```

Comment:
We shouldn't need to add this explicitly bc it's part of `BoxProps`. Was something not working?